app/controllers/api/v1/rents_controller.rb

```diff
@@ -19,6 +21,10 @@ def create
       def rent_params
         params.require(:rent).permit(:book_id, :user_id, :from, :to)
       end
+
+      def set_locale
+        I18n.locale = current_user.locale || I18n.default_locale
```

Comment:
I don't think this should be done in the `RentsController` I would do it in the `ApplicationController`